src/object/camera.cpp

```diff
@@ -319,7 +319,7 @@ Camera::update(float dt_sec)
       {
         update_scroll_normal_multiplayer(dt_sec);
       }
-      else
+      else if (Sector::current() && !(Sector::current()->get_players()[0]->is_dying()))
```

Comment:
Seeing the `if` above this also checks for `Sector::current()`, both statements could be nested in an `if (Sector::current())` to not check twice.